synaptic_reconstruction/inference/util.py

```diff
@@ -44,14 +44,14 @@ def get_prediction(
     Returns:
         The predicted volume.
     """
-    # We always use the same default halo.
-    halo = {"x": 64, "y": 64, "z": 16}
```

Comment:
This was never used, so I removed it. I think you made the corresponding changes below already.